examples/express/index.ts

```diff
@@ -145,7 +145,10 @@ apiRouter.get('/userinfo', async (req, res) => {
   return res.json(userinfo)
 })
 
-apiRouter.get('/logout', async (_req, res) => {
+apiRouter.get('/logout', async (req, res) => {
+  const sessionId = String(req.cookies[SESSION_COOKIE_NAME])
+  // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
```

Comment:
Since `Map` is not fully supported by all browsers, I chose to turn off the rule instead. fixed in [beef837](https://github.com/opengovsg/sgid-client/pull/50/commits/beef837481ad8aa7430287c2c203bfe6055b1e07)